splinepy/helpme/create.py

```diff
@@ -298,6 +299,188 @@ def revolved(
     return type(spline)(**spline_dict)
 
 
+def swept(
+    cross_section,
+    trajectory,
+    nsections=None,
+    cross_section_normal=None,
+):
+    """Sweeps a cross-section along a trajectory
```

Comment:
Can you please add details here?

Reply:
Added some; maybe you can check whether it's sufficient.

Reply:
I meant what's implemented: reference to algorithm / equation number from the NURBS book. If applicable, you can directly add them here see Mapper's notes for format

Reply:
Added some references in the code